### PR TITLE
Crossbreed Descriptions

### PR DESCRIPTION
**Pull request by Ty-the-Smonk**
## About The Pull Request
Adds descriptions to what each crossbreed does when examined.

## Why It's Good For The Game
Mostly for sanity's sake, newer players, other people learning xenobio, or people that have just murdered the xenobiologist for his slimy bullshit won't have to look at the wiki as much or if they just wanna screw around and see what does what without a trial by fire.

## Changelog
:cl:
add: Descriptions for every crossbreed's effect (minus recurring because its self explanatory).
fix: Changes a single comment in _corecross to reflect how many extracts are needed to crossbreed.
/:cl:

### CODE REVIEW

**Comment by knacker48** [comment]
Looks handy. Just keep in mind that #43870 is getting merged so you'll probably have to change the description on the regenerative sepia crossbreed

**Comment by Ty-the-Smonk** [comment]
@knacker48 Its basically the same it just tweaked the deja vu effect

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 1]
```diff
@@ -164,20 +174,23 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/pyrite
 	colour = "pyrite"
+	effect_desc = "Fully heals the target and randomly colors the target."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Fully heals and randomly colors the target.

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 2]
```diff
@@ -164,20 +174,23 @@ Regenerative extracts:
...
 
 /obj/item/slimecross/regenerative/pyrite/core_effect(mob/living/target, mob/user)
 	target.visible_message("<span class='warning'>The milky goo coating [target] leaves [target.p_them()] a different color!</span>")
 	target.add_atom_colour(rgb(rand(0,255),rand(0,255),rand(0,255)),WASHABLE_COLOUR_PRIORITY)
 
 /obj/item/slimecross/regenerative/red
 	colour = "red"
+	effect_desc = "Fully heals the target and injects the target with some ephedrine."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Fully heals the target and injects them with some ephedrine.

I don't like having to use "them" but we often have to.

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 3]
```diff
@@ -190,13 +203,15 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/pink
 	colour = "pink"
+	effect_desc = "Fully heals the target and injects the target with some krokodil."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: the target -> them

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 4]
```diff
@@ -214,6 +230,7 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/black
 	colour = "black"
+	effect_desc = "Fully heals the target and creates a duplicate of the target, that drops dead soon after."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: the target -> them

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 5]
```diff
@@ -244,12 +262,14 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/adamantine
 	colour = "adamantine"
+	effect_desc = "Fully heals the target and boosts the target's armor."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Fully heals the target and boosts their armor temporarily.

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 6]
```diff
@@ -244,12 +262,14 @@ Regenerative extracts:
...
 
 /obj/item/slimecross/regenerative/adamantine/core_effect(mob/living/target, mob/user) //WIP - Find out why this doesn't work.
 	target.apply_status_effect(STATUS_EFFECT_SLIMESKIN)
 
 /obj/item/slimecross/regenerative/rainbow
 	colour = "rainbow"
+	effect_desc = "Fully heals the target and temporarily makes the user immortal, yet pacifistic."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Fully heals the target and temporarily makes them immortal, but pacifistic.

File: code/modules/research/xenobiology/crossbreeding/reproductive.dm [hunk 7]
```diff
@@ -8,6 +8,7 @@ Reproductive extracts:
 	desc = "It pulses with a strange hunger."
 	icon_state = "reproductive"
 	effect = "reproductive"
+	effect_desc = "When fed monkey cubes it produces more extracts, bio bag compatible as well."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/reproductive.dm]: period and new sentence after "more extracts".

File: code/modules/research/xenobiology/crossbreeding/selfsustaining.dm [hunk 8]
```diff
@@ -12,6 +12,7 @@ Self-sustaining extracts:
 	name = "autoslime"
 	desc = "It resembles a normal slime extract, but seems filled with a strange, multi-colored fluid."
 	var/obj/item/slime_extract/extract
+	var/effect_desc = "Creates one to four extracts that, when used, lets you choose what reaction you want."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/selfsustaining.dm]: Can change entire description to:
A self-sustaining slime extract. When used, lets you choose which reaction you want.

The autoslime is the end-product. Self-sustaining has no pre-item like burning. Also always creates 4 autoslimes.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 9]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
+
+
 //Colors and subtypes:
 /obj/item/slimecross/stabilized/grey
 	colour = "grey"
+	effect_desc = "Makes slimes friendly to the owner"
 
 /obj/item/slimecross/stabilized/orange
 	colour = "orange"
+	effect_desc = "Passively ties to increase or decrease the owner's body temperature to normal"
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: tries

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 10]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 	colour = "purple"
+	effect_desc = "Provides a regeneration effect"
 
 /obj/item/slimecross/stabilized/blue
 	colour = "blue"
+	effect_desc = "Makes the owner immune to slipping on water, soap or foam. Space lube and ice are still too slippery."
 
 /obj/item/slimecross/stabilized/metal
 	colour = "metal"
+	effect_desc = "Adds a sheet of material to a stack in the owners backpack, it is based on a timer."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Every 30 seconds, adds a sheet of material to a random stack in the owner's backpack.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 11]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/yellow
 	colour = "yellow"
+	effect_desc = "Every couple of seconds it recharges a device on the owner by 10%."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: couple of seconds -> ten seconds or few seconds

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 12]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/darkpurple
 	colour = "dark purple"
+	effect_desc = "Gives the owner burning finger tips."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Gives you burning fingertips, automatically cooking any microwavable food you hold.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 13]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/darkblue
 	colour = "dark blue"
+	effect_desc = "Slowly extinguishes the owner if they are on fire, also wets held items."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: You can remove " , also wets held items."

Maybe it "extinguishes"? Wets is misleading either way.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 14]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/silver
 	colour = "silver"
+	effect_desc = "Slows the rate at which the owner loses nutrition"
 
 /obj/item/slimecross/stabilized/bluespace
 	colour = "bluespace"
+	effect_desc = "On a two minute cooldown, when the owner has taken enough damage, they are teleported a place safe for humans."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: to a safe place

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 15]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/sepia
 	colour = "sepia"
+	effect_desc = "Randomly adjusts the owner's speed."
 
 /obj/item/slimecross/stabilized/cerulean
 	colour = "cerulean"
+	effect_desc = "Creates a duplicate of the owner, if the owner dies they will take contol of the duplicate, unless the death was by beheading or gibbing."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Period and new sentence after first "owner". 

contol -> control
by -> from

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 16]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 	colour = "pyrite"
+	effect_desc = "Randomly colors the owner every few seconds."
 
 /obj/item/slimecross/stabilized/red
 	colour = "red"
+	effect_desc = "Nullifies all equipment based slowdowns."
 
 /obj/item/slimecross/stabilized/green
 	colour = "green"
+	effect_desc = "Changes the owner's name and appearence while holding this extract."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: appearence ->  appearance

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 17]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/pink
 	colour = "pink"
+	effect_desc = "As long as no creatures are harmed in the owner's presense, they will not attack you, if the peace is broken, takes two minutes to restore."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: period and new sentence before "If the peace is broken".

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 18]
```diff
@@ -142,18 +161,23 @@ Stabilized extracts:
...
 	colour = "oil"
+	effect_desc = "The owner will violently explode when they die while holding this extract."
 
 /obj/item/slimecross/stabilized/black
 	colour = "black"
+	effect_desc = "While strangling someone, the owner's hands melt around their neck, draining their life in exchange for food and healing."
 
 /obj/item/slimecross/stabilized/lightpink
 	colour = "light pink"
+	effect_desc = "The owner moves at high speeds while holding this extract, also stabilizes anyone in critical condition around you using Epinephrine."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: period and new sentence before "Also". 

Lowercase epinephrine.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 19]
```diff
@@ -142,18 +161,23 @@ Stabilized extracts:
...
 
 /obj/item/slimecross/stabilized/adamantine
 	colour = "adamantine"
+	effect_desc = "Owner gains a slight boost in damage resistance to all types."
 
 /obj/item/slimecross/stabilized/rainbow
 	colour = "rainbow"
+	effect_desc = "Accepts a regenerative extract, automatically uses it if the owner goes into a critical condition."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Accepts a regenerative extract and automatically uses it if the owner enters a critical condition.

**Comment by Ty-the-Smonk** [comment]
@Angustmeta oof ill work on this in a bit

**Comment by Angustmeta** [comment]
Next time I'll learn to use the request changes feature instead of review comments.

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 20]
```diff
@@ -126,13 +132,15 @@ Regenerative extracts:
...
 	colour = "silver"
+	effect_desc = "Fully heals the target and makes the target's belly feel round and full."
 
 /obj/item/slimecross/regenerative/silver/core_effect(mob/living/target, mob/user)
 	target.set_nutrition(NUTRITION_LEVEL_FULL - 1)
 	to_chat(target, "<span class='notice'>You feel satiated.</span>")
 
 /obj/item/slimecross/regenerative/bluespace
 	colour = "bluespace"
+	effect_desc = "Fully heals the target and teleports the target to where it was created."
```
[the remark this answers is not in the document]
Comment (Ty-the-Smonk) [reply code/modules/research/xenobiology/crossbreeding/regenerative.dm]: I dont know why i was so afraid of their and them

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm (hunk 13 above)
Comment (Ty-the-Smonk) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm]: On the wiki it says wets so i dunno

Comment (Ty-the-Smonk) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Ok so I looked in the code and it auto makes things like dehydrated carps and monkey cubes produce the mob

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 21]
```diff
@@ -66,35 +66,35 @@ Stabilized extracts:
 
 /obj/item/slimecross/stabilized/metal
 	colour = "metal"
-	effect_desc = "Adds a sheet of material to a stack in the owners backpack, it is based on a timer."
+	effect_desc = "Every 30 seconds, adds a sheet of material to a random stack in the owner's backpack."
 
 /obj/item/slimecross/stabilized/yellow
 	colour = "yellow"
-	effect_desc = "Every couple of seconds it recharges a device on the owner by 10%."
+	effect_desc = "Every ten seconds or few seconds it recharges a device on the owner by 10%."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Pardon me, I meant to say choose between "ten seconds" and "few seconds". Just go with "ten seconds" imo.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 22]
```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
...
 	colour = "blue"
+	effect_desc = "Makes the owner immune to slipping on water, soap or foam. Space lube and ice are still too slippery."
 
 /obj/item/slimecross/stabilized/metal
 	colour = "metal"
+	effect_desc = "Every 30 seconds, adds a sheet of material to a random stack in the owner's backpack."
 
 /obj/item/slimecross/stabilized/yellow
 	colour = "yellow"
+	effect_desc = "Every ten seconds or few seconds it recharges a device on the owner by 10%."
```
Comment (Angustmeta) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm]:
```suggestion
	effect_desc = "Every ten seconds it recharges a device on the owner by 10%."
```

File: code/modules/research/xenobiology/crossbreeding/burning.dm [hunk 23]
```diff
@@ -29,6 +29,7 @@ Burning extracts:
 
 /obj/item/slimecross/burning/grey
 	colour = "grey"
+	effect_desc = "It creates a hungry and speedy slime that will love you forever."
```
Comment (4dplanner) [inline code/modules/research/xenobiology/crossbreeding/burning.dm]: I feel that the alternating between "It creates..." and "Creates..." is awkward

File: code/modules/research/xenobiology/crossbreeding/burning.dm [hunk 24]
```diff
@@ -53,6 +55,7 @@ Burning extracts:
 
 /obj/item/slimecross/burning/purple
 	colour = "purple"
+	effect_desc = "A clump of invigorating gel is created after use, it has healing properties and makes you feel good."
```
Comment (4dplanner) [inline code/modules/research/xenobiology/crossbreeding/burning.dm]: A third style? I think it'd be better if it was consistently "Creates a clump of invigorating gel".

File: code/modules/research/xenobiology/crossbreeding/burning.dm [hunk 25]
```diff
@@ -40,6 +41,7 @@ Burning extracts:
 
 /obj/item/slimecross/burning/orange
 	colour = "orange"
+	effect_desc = "Expels pepperspray in a radius when activated, it can also effect you!"
```
Comment (4dplanner) [inline code/modules/research/xenobiology/crossbreeding/burning.dm]: Example of the second style, though awkwardly uses it after the comma

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm [hunk 26]
```diff
@@ -126,13 +132,15 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/silver
 	colour = "silver"
+	effect_desc = "Fully heals the target and makes their belly feel round and full."
```
Comment (4dplanner) [inline code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Christ you did a lol of these lol

**Review by 4dplanner — COMMENTED** [review]
Basically needs some more consistency. I think the "Expels a powergamer" style is best.

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm (hunk 22 above)
Comment (Ty-the-Smonk) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm]: Rad

File: code/modules/research/xenobiology/crossbreeding/regenerative.dm (hunk 26 above)
Comment (Ty-the-Smonk) [reply code/modules/research/xenobiology/crossbreeding/regenerative.dm]: Yes I did a lol of these

**Pull request merged**